Makefile

```diff
@@ -49,7 +49,7 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 endif
 
 # Image URL to use all building/pushing image targets
-IMG ?= $(IMAGE_TAG_BASE):$(VERSION)
```

Comment:
Do you know if the skeleton generated by the latest operator-sdk makes this change ?
If not, will it break bundle and catalog generation if there is an implicit assumption to use version without `v` prefix ?

Reply:
it seems the generated skeleton suggests to push this image to "latest", hence, i assumed this was changed locally and it's fine to change it again

Reply:
Operator-sdk introduced the `v` in the bundle and catalog versioning in v1.6.0 with https://github.com/operator-framework/operator-sdk/pull/4406. Specifically this intermediate change in the PR https://github.com/operator-framework/operator-sdk/compare/9ffbb583609404fa3ab27d59c1e1c624255ad8aa..f8a08b782a90e7a1af8493bdaa9693bc61692dd9#diff-97e56898a92a31cc551cc061b2eb64809ad7fc45c2bacd13c660ae207632538fL139

I've checked the review comments that preceded this change but I don't find any suggestion about the extra `v`... Maybe worth asking operator-sdk folks to sort this out.

Reply:
I could chat with an operator-sdk developer on [slack](https://kubernetes.slack.com/archives/C017UU45SHL/p1700562561803609). The outcome is that the `v` was introduced for _convenience_, for whatever that means, and we can do pretty much what we want.

Reply:
@bpradipt can we merge this change ?

Reply:
sure..